ansible/ansible.cfg

```diff
@@ -1,5 +1,5 @@
 [defaults]
-stdout_callback=selective
+stdout_callback=debug
```

Comment:
should be reverted before merge

Reply:
agree , will revert when we are going to merge it